server/group/API/Tasks/createdEditTasks.js

```diff
@@ -0,0 +1,12 @@
+const { editTask } = require("../../../Base/Task/editTask")
+const { insertTable } = require("../../../Base/Task/insertTask")
+
+function createdEditTasks(app, client) {
+    app.route('/task').post(async (req,res) => {
+        let result = await insertTable(client, req.body, req.session.userLogin)
+        res.send('complete')
+    }).put(async (req,res) => {
+            let result = await editTask(client, req.body)
+            res.send('complete')
+      })
```

Comment:
Better to use routers for each entity (user, task, e.t.c) like in project which I sent you.
```
const { Router } = require('express')

function createdEditTasks(app, client) {
  const taskRouter = Router()
  app.use('/task', taskRouter)

  taskRouter.post(async (req,res) => {
      let result = await insertTable(client, req.body, req.session.userLogin)
      res.send('complete')
  })

  taskRouter.put(async (req,res) => {
      let result = await editTask(client, req.body)
      res.send('complete')
  })
}

```